.github/ISSUE_TEMPLATE/spec_request.md

```diff
@@ -0,0 +1,113 @@
+* ... discuss implementation details?
+* ... just dump info useful for implementing the spec?
+
+Then this template is for you.
+
+Since for every format we deal with the same info and since it is beneficial for both human and bot (one is planned!) readers to have the info encoded into a messages with the same schema, we have designed the format for this kind of issues. Please, follow it.
+
+A request for a format specification consists of some YAML blocks followed by a free-form text description. Each YAML block has own purpose, its purpose is described in the comment within that block. 
+
+We expect you to:
+* fill in all the fields you can and want to fill in;
+* delete all the unfilled fields;
+* delete all the comments (both YAML and HTML) from this template (including this one!).
+* make sure that YAML blocks contain well-formed YAML;
+* make sure that blocks contents do not violate their schemas;
+* make sure that all the fields marked as MANDATORY are filled in.
+
+-->
+```yaml
+# The purposes of this block are:
+# * storing the info uniquily identifying the format to make it searcheable;
+# * storing useful information about the format in a convenient structured way;
+# * providing an implementer with a boilerplate which can be just copied into a KSY file;
+# The content of this block must be a part of a valid KSY.
+# This block must be present.
+
+meta: # see https://doc.kaitai.io/ksy_reference.html#meta for more info
+  id: # insert an ID (matches the regexp [a-z][\da-z_]*[\da-z]) you feel right to assign to the format here. KS currently has no namespaces, so the id MUST be unique within all the repo and you should be sure there will be no collisions. If in doubt, add a prefix of an app name. MANDATORY.
+  title: # insert a human-readable name of the format here. MANDATORY.
+  application: # insert a human-readable name of the application for which this format is native here. If it is native for several applications, choose the ones you feel the most affiliated with the format. If the format is a widespread one, like PNG or WAV, just remove this field.
+  file-extension:
+    # insert here a YAML list of file extensions should be placed here.
+  xref: # check if the following sources contain info about the format you request. See https://doc.kaitai.io/user_guide.html#meta-xref for more info
+    din:
+      id: # DIN standard identifier
+    forensicswiki: # Page name on https://forensicswiki.org/
+    fileformat: # Link to https://www.fileformat.info
+    iso: # ISO standard identifier
+    justsolve: # ID on http://fileformats.archiveteam.org/wiki/
+    loc: # ID in https://www.loc.gov/preservation/digital/formats/
+    mime: # MIME type
+    pronom: # IDs on https://www.nationalarchives.gov.uk/pronom/fmt/
+     - x-fmt/
+    rfc: # Link to RFC
+    wikidata: Q # ID on WikiData
+
+doc: |
+  # * Describe your format briefly. MANDATORY
+  # * add links to the sources of sample files in the Web. MANDATORY
```

Comment:
I wouldn't say that "sources of sample files in the Web" are mandatory. I haven't seen any spec in ksf that actually does this. (Don't get me wrong, I think this would be useful, but it shouldn't be mandatory.)

Reply:
Yeah, they are not very mandatory.